File: SauceDemo/PageObjects/CheckoutPage2.cs
using OpenQA.Selenium;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace SeleniumAutomation.PageObjects
{
    public class CheckoutPage2 : BasePage
    {

        public CheckoutPage2(IWebDriver driver) : base(driver)
        {
        }

        // I'm doing it this way to show a quick way to get all the inputs as webelements.
        private IWebElement BUT_finish => HelperMethods.FindElementWithWait_Clickable(_driver, By.XPath("//*[@data-test='finish']"), TimeSpan.FromSeconds(10));

        //I'm then going to populate the inputs from a list of data in the test
        public void ConfirmOrderInfo()
        {
            BUT_finish.Click();
        }

        public void gotoCheckoutPage2()
        {
            Goto("checkout-step-two.html");
        }
    }
}

File: Selenium_Python/pythonProject/pages/checkoutpage2.py
from selenium.webdriver.common.by import By
from selenium.webdriver.support.ui import WebDriverWait
from selenium.webdriver.support import expected_conditions as EC
from selenium.common.exceptions import NoSuchElementException

from helper import helper
from pages.basepage import basepage

class checkoutpage2(basepage):
    def __init__(self, driver):
        super().__init__(driver)
        self.but_finish = helper.find_element_with_wait_clickable(self._driver, By.XPATH, "//*[@data-test='finish']", 20)

    def confirm_order_info(self):
        self.but_finish.click()

    def goto_checkout_page2(self):
        self.goto("checkout-step-two.html")
